code/modules/mob/living/living.dm

```diff
@@ -357,6 +357,7 @@
 		updatehealth() //then we check if the mob should wake up.
 		update_canmove()
 		update_sight()
+		clear_alert("oxy")
```

Comment:
It isn't, evidently. For example, put someone in crit, turn them into an ash lizard, then revive them, they'll have an O2 alert on their screen forever.